packages/components/src/shortened-number/index.tsx

```diff
@@ -0,0 +1,29 @@
+export type ShortenedNumberProps = {
+	value: number | null;
+};
+
+const LOCALE = ( typeof window === 'undefined' ? null : window.navigator?.language ) ?? 'en-US';
+const FORMATTER = new Intl.NumberFormat( LOCALE, {
+	notation: 'compact',
+	compactDisplay: 'short',
+	maximumFractionDigits: 1,
+} );
+
+function formatNumber( value: number | null ) {
+	if ( ! Number.isFinite( value ) ) {
+		return '-';
+	}
+
+	return Number.isFinite( value ) ? FORMATTER.format( value as number ) : '-';
```

Comment:
```suggestion
	return FORMATTER.format( value as number );
```

Could this line be simplified to return the formatted value only?

Reply:
D'oh, good catch. Fixed.